### DISCUSSION TITLE
!!MISSING command!! message in my rust files

### DISCUSSION THREAD

**Discussion by 01speed1** [rust-lang/rust-analyzer#14569] [General]
Hey guys!
I don't know why I'm getting this message, maybe I change something?, please help!, let me know if you need to see my configuration or something

![image](https://user-images.githubusercontent.com/7319582/231960043-4805bb77-d759-4c02-a51c-0f20ea272b24.png)

what I have touched
![image](https://user-images.githubusercontent.com/7319582/231960943-0bfcea98-25c3-47ad-a2d5-b09164ef525c.png)

**Comment by lnicola** [comment]
Duplicate of #14568.
